script/upgrades/dependencies.json

```diff
@@ -29,8 +35,7 @@
     "Chainlink.AUDUSD": "0xf2Bd4FAa89f5A360cDf118bccD183307fDBcB6F5",
     "Chainlink.CADUSD": "0x2f6d6cB9e01d63e1a1873BACc5BfD4e7d4e461d1",
     "Chainlink.CHFUSD": "0xfd49bFcb3dc4aAa713c25e7d23B14BB39C4B8857",
-    "Chainlink.JPYUSD": "0xf323563241BF8B77a2979e9edC1181788A98EcB2",
```

Comment:
double check if these should be deleted (prolly not :D)

Reply:
readded